### ISSUE TITLE
Sub-optimal Percona aggregation.

### ISSUE THREAD

**Issue by jfg956** [oursqlcommunity-org/planet#68]
The Percona Community Blog has too much non-MySQL content.  Examples:
- https://percona.community/blog/2022/12/13/how-to-generate-data-with-pagila-in-percona-distribution-for-postgresql/

And the Percona Database Performance Blog mysql-and-variants does not have all interesting posts.  Example missing:
- https://www.percona.com/blog/elevate-your-percona-live-experience-by-climbing-mt-elbert/
- https://www.percona.com/blog/release-roundup-january-9-2022/

**Comment by jfg956** [comment]
Suggested solution in this PR: https://github.com/oursqlcommunity-org/planet/pull/69

**Comment by dbazhenov** [comment]
I asked the blog manager to check and he said that the posts from the issue were not tagged "mysql-and-variants".
Anyway, thanks for pointing that out, we'll try to keep a closer eye on tags and RSS.

**Comment by jfg956** [comment]
Thanks for your reply Daniil.

The problems for the Percona Performance Blog and the Percona Community Blog were different:
* Percona Community Blog: there were no tagged feed since the migration to new hosting [1], so Postgres content was reaching the aggregator,
* Percona Performance Blog: some interesting content was not tagged with the `mysql-and-variants`.

[1]: https://github.com/oursqlcommunity-org/planet/commit/e395ed892f9ce0c4441a66b5c058481db1c64c81

What is your preferred way forward:
1. provide a tagged feed for the Percona Community Blog, and we continue with the `mysql-and-variants` tag for the Percona Performance Blog,
2. we go forward with rss.app filtering.

Note that I was using tagged feed in the past because I did not have a filtering solution.  My suggestion to use filtering now is because I recently integrated filtering with rss.app.
